src/components/Pagination/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import IconButton from '../IconButton';
+import { theme } from '../../theme';
+
+const Pagination = props => {
+  // Don't show it if both are disabled
+  if (props.prevDisabled && props.nextDisabled) {
+    return null;
+  }
+
+  const { translations } = props;
```

Comment:
Please use the same pattern as in the Uploader component
https://github.com/heetch/flamingo/blob/master/src/components/Uploader/index.js#L43

Reply:
🆙 
`translate` is a function returning an `id` and a fallback
The goal would be to use it as `translate={t}` to translate the whole component, where `t` can be pulled from `i18next` for example